NetworkingTests/Tests/EntityRepositorySpec.swift

```diff
@@ -154,6 +191,23 @@ class EntityRepositorySpec: QuickSpec {
             
         }
         
+        describe("#fetchCustomFailingEntityMishandlingError") {
+            
+            it("fetches a single entity from JSON file and fails with a custom error mishandled") { waitUntil { done in
+                repository.fetchCustomFailingEntityMishandlingError().startWithResult {
+                    switch $0 {
+                    case .success: fail()
+                    case .failure(let error):
+                        switch error {
+                        case .requestError: done()
```

Comment:
Then I would change the context to:
```
context("when the request error doesn't map any custom error") {
    it("fetches a single entity from JSON file and fails with a request error") {
        ...
    }
}
```
And I would add an example of mapping to a custom error.